docs/csharp/whats-new/tutorials/primary-constructors.md

```diff
@@ -1,71 +1,76 @@
-C# 12 introduces [*primary constructors*](../../programming-guide/classes-and-structs/instance-constructors.md#primary-constructors), a concise syntax to declare constructors whose parameters are available anywhere in the body of the type.
+C# 12 introduces [primary constructors](../../programming-guide/classes-and-structs/instance-constructors.md#primary-constructors), which provide a concise syntax to declare constructors whose parameters are available anywhere in the body of the type.
 
-In this tutorial, you will learn:
-
-> [!div class="checklist"]
->
-> - When to declare a primary constructor on your type
-> - How to call primary constructors from other constructors
-> - How to use primary constructor parameters in members of the type
-> - Where primary constructor parameters are stored
+This article describes how to declare a primary constructor on your type and recognize where to store primary constructor parameters. You can call primary constructors from other constructors and use primary constructor parameters in members of the type.
 
 ## Prerequisites
 
 [!INCLUDE [Prerequisites](../../../../includes/prerequisites-basic.md)]
 
-## Primary constructors
+## Understand rules for primary constructors
+
+You can add parameters to a `struct` or `class` declaration to create a *primary constructor*. Primary constructor parameters are in scope throughout the class definition. It's important to view primary constructor parameters as *parameters* even though they are in scope throughout the class definition.
 
-You can add parameters to a `struct` or `class` declaration to create a *primary constructor*. Primary constructor parameters are in scope throughout the class definition. It's important to view primary constructor parameters as *parameters* even though they are in scope throughout the class definition. Several rules clarify that they're parameters:
+Several rules clarify that these constructors are parameters:
 
-1. Primary constructor parameters may not be stored if they aren't needed.
-1. Primary constructor parameters aren't members of the class. For example, a primary constructor parameter named `param` can't be accessed as `this.param`.
-1. Primary constructor parameters can be assigned to.
-1. Primary constructor parameters don't become properties, except in [`record`](../../language-reference/builtin-types/record.md) types.
+- Primary constructor parameters can't be stored if they aren't needed.
+- Primary constructor parameters aren't members of the class. For example, a primary constructor parameter named `param` can't be accessed as `this.param`.
+- Primary constructor parameters can be assigned to.
+- Primary constructor parameters don't become properties, except in [record](../../language-reference/builtin-types/record.md) types.
 
-These rules are the same as parameters to any method, including other constructor declarations.
+These rules are the same for parameters to any method, including other constructor declarations.
```

Comment:
I want to rephrase this, to emphasize that it's consistent with existing features:

```suggestion
These rules are the same rules already defined for parameters to any method, including other constructor declarations.
```